src/workerd/jsg/async-context.h

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+#pragma once
+
+#include "jsg.h"
+#include <v8.h>
+
+namespace workerd::jsg {
+
+class AsyncResource {
```

Comment:
What is a "resource"? (Please explain in the comment.)

Reply:
I guess you can refer to the readme, but I feel like the text there also doesn't really introduce this term "resource" but assumes I already know what it means.

Reply:
Expanded the docs comments a bit more.